app/views/users/show.html.erb

```diff
@@ -1 +1,14 @@
-<h1>User Profile</h1>
+<main>
+  <section>
+    <ul>
+      <% @user.recent_posts.each do |post| %>
+        <%= link_to user_user_post_path(@user, post) do %>
+          <li class="card flexColumn hover">
+            <h3 class="left"><%= post.title %></h3>
+            <h4 class="left"><%= post.text %></h4>
+          </li>
+        <% end %>
+      <% end %>
+    </ul>
```

Comment:
- [ ] There should be a button that takes a user to the "posts index page"(/users/1/posts) called "See All Posts" as seen in the provided wireframe
![image](https://github.com/MarcoDDM/blogapp/assets/45460192/7786e2b0-e1a2-42ac-aa89-00e0035e609f)